show-portal-user-info/src/main/java/com/esri/arcgismaps/sample/showportaluserinfo/components/AuthenticationAppViewModel.kt

```diff
@@ -0,0 +1,119 @@
+        // create your own client ID. For more info see: https://developers.arcgis.com/documentation/mapping-apis-and-services/security/tutorials/register-your-application/
+        "rR5K4TEyhkPDcTkf",
+        "my-ags-app://auth"
+    )
+
+    private val _portalUserName = MutableStateFlow(String())
+    val portalUserName: StateFlow<String> = _portalUserName.asStateFlow()
+
+    private val _emailID = MutableStateFlow(String())
+    val emailID: StateFlow<String> = _emailID.asStateFlow()
+
+    private val _userCreationDate = MutableStateFlow(String())
+    val userCreationDate: StateFlow<String> = _userCreationDate.asStateFlow()
+
+    private val _portalName = MutableStateFlow(String())
+    val portalName: StateFlow<String> = _portalName.asStateFlow()
+
+    private val defaultBitmap = BitmapFactory.decodeResource(application.resources, R.drawable.user)
+
+    private val _userThumbnail: MutableStateFlow<Bitmap?> = MutableStateFlow(defaultBitmap)
+    val userThumbnail: StateFlow<Bitmap?> = _userThumbnail.asStateFlow()
+
+    private val _infoText: MutableStateFlow<String> = MutableStateFlow(startInfoText)
+    val infoText: StateFlow<String> = _infoText.asStateFlow()
+
+    private val _isLoading: MutableStateFlow<Boolean> = MutableStateFlow(false)
+    val isLoading: StateFlow<Boolean> = _isLoading.asStateFlow()
+
+    private val _url: MutableStateFlow<String> = MutableStateFlow(arcGISUrl)
+    val url: StateFlow<String> = _url.asStateFlow()
+    fun setUrl(newUrl: String) {
+        _url.value = newUrl
+    }
+
+    fun signOut() = viewModelScope.launch {
+        _isLoading.value = true
+        ArcGISEnvironment.authenticationManager.signOut()
+        _infoText.value = startInfoText
+        _isLoading.value = false
+        _portalUserName.value = ""
+        _emailID.value = ""
+        _userCreationDate.value = ""
+        _portalName.value = ""
+    }
+
+    fun loadPortal() = viewModelScope.launch {
+        _isLoading.value = true
+        authenticatorState.oAuthUserConfiguration = oAuthUserConfiguration
+        val portal = Portal(url.value, Portal.Connection.Authenticated)
+        portal.load().also {
+            _isLoading.value = false
+        }.onFailure {
+            _infoText.value = it.toString()
+        }.onSuccess {
+            portal.portalInfo?.apply {
+                _portalUserName.value = this.user?.fullName ?: noPortalInfoText
+                _emailID.value = this.user?.email ?: noPortalInfoText
+                _portalName.value = this.portalName ?: noPortalInfoText
+                // get the created date
+                val date: Date? = Date.from(this.user?.creationDate)
+                val formatter = SimpleDateFormat("dd-MMM-yyyy", Locale.US)
+                val formattedDate: String = formatter.format(date)
+                _userCreationDate.value = formattedDate
+                _userThumbnail.value = this.user?.thumbnail?.image?.bitmap ?: defaultBitmap
```

Comment:
When I log in I do not see a thumbnail loaded. I have a profile picture, and it loads on the v100 sample. In v200 the thumbnail is now a [LoadableImage](https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.portal/-portal-user/thumbnail.html), which would mean we need to load the thumbnail. But when I try this I get an error: 

```kotlin
val uri = user.thumbnail.uri // "blob.png"
val loadedThumbnail = user?.thumbnail?.load() // returns null
// Error: Expected URL scheme 'http' or 'https' but no scheme was found for blob.png
```

@prupani-7 Are you able to load and display a thumbnail here? Or does it seem like an Authenticator issue @hud10837?

Reply:
@shubham7109 good catch! I tried similar code below to load the image, 

```
this.user?.thumbnail?.load()?.onSuccess {
                    _userThumbnail.value = this.user?.thumbnail?.image?.bitmap
                }?.onFailure {
                   it.message.toString()
                   it.stackTraceToString()
                   println("Image not loaded")
                   }
                
```
I get the same error: 
`Failure(java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no scheme was found for blob.p...`

StackTrace:
```
java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no scheme was found for blob.p...
	at okhttp3.HttpUrl$Builder.parse$okhttp(HttpUrl.kt:1261)
	at okhttp3.HttpUrl$Companion.get(HttpUrl.kt:1634)
	at com.arcgismaps.http.internal.ExtensionsKt.toOkHttp(Extensions.kt:160)
	at com.arcgismaps.http.internal.HttpClientImpl.awaitResponse(HttpClientImpl.kt:164)
	at com.arcgismaps.http.internal.HttpClientImpl.execute-gIAlu-s(HttpClientImpl.kt:143)
	at com.arcgismaps.httpcore.ArcGISHttpClient.fetchResponse-gIAlu-s$api_release(ArcGISHttpClient.kt:135)
	at com.arcgismaps.http.internal.RtcRequestHandler$handleRequest$2$1.invokeSuspend(RtcRequestHandler.kt:73)
	at com.arcgismaps.http.internal.RtcRequestHandler$handleRequest$2$1.invoke(Unknown Source:8)
	at com.arcgismaps.http.internal.RtcRequestHandler$handleRequest$2$1.invoke(Unknown Source:4)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:264)
	at com.arcgismaps.http.internal.RtcRequestHandler.handleRequest(RtcRequestHandler.kt:60)
	at com.arcgismaps.ArcGISEnvironment$onRequestRequired$1.invokeSuspend(ArcGISEnvironment.kt:349)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```

Reply:
This seems like an issue. Recently [similar issue](https://devtopia.esri.com/runtime/kotlin/issues/2498) with ProtalGroup thumbnail was resolved. 
Issue created here: https://devtopia.esri.com/runtime/kotlin/issues/3029